.gitignore

```diff
@@ -1,3 +1,3 @@
 node_modules/
 coverage/
-lib/
+.idea/
```

Comment:
Please use a global gitignore for your environment

**~/.gitconfig**
```
[core]
	excludesfile = ~/.gitignore_global
```

**~/.gitignore_global**
```
.idea/
```

Reply:
oops :) its fixed :)